compiler/rustc_interface/src/interface.rs

```diff
@@ -177,6 +180,10 @@ pub fn create_compiler_and_run<R>(config: Config, f: impl FnOnce(&Compiler) -> R
         registry.clone(),
     );
 
+    if let Some(parse_sess_created) = config.parse_sess_created {
+        parse_sess_created(&mut Lrc::get_mut(&mut sess).unwrap().parse_sess);
```

Comment:
Please use `.expect` with a message informing us that we are guaranteed to still have unique access to the session here.

I guess it could be cleaner to make `create_session` return the owned session instead of the reference counted one, but that may be a bit more invasive than useful for this PR

Reply:
Done. :+1:

@rustbot label -S-waiting-on-bors S-waiting-on-review